docs/README.md

```diff
@@ -183,19 +183,93 @@ end
 You can also override an entire controller by simply placing a file with the controller's name in the controllers directory.  For example, `app/controllers/general_controller.rb` would take the place of the Orchid version of this file.  *This approach is not recommended.*
 
 ### Facets
-You may also want to peruse the `app/models/facets.rb` file and alter it for specific fields which you would prefer.  If your app supports multiple languages, you will need to make sure that the facets are organized by language, for example:
+Facets are API fields used to index items in the "Browse" pages. For each
+language, facets' text labels and options may be redefined and each facet may
+also optionally:
 
+- Function as a search filter
+- Have its values from the API translated
+
+Facet configuration is defined in `config/public.yml` for each language as:
+```yaml
+…
+  facets:
+    [language]:
+      [api_field]:
+        label: [display_text]
+        # Optional features
+        flags:
+          - search_filter
+          - translate
+…
 ```
-{
-  "en": {
-    ...facets...
-  },
-  "es": {
-    ...facets...
-  }
-}
+
+- `language` - Language code matching the code(s) set in `config/public.yml`
+- `api_field` - Field name from the API. Field names may be nested (e.g.
+  `creator.name`)
+- `display_text` - Text displayed for the facet label for this language
+
+The `flags:` key and its list of contained flag names may be omitted. A flag's
+presence in the config file enables changes in facet use and behavior as
+follows:
+
+`search_filter` - Enables use of facets as search filters on search pages.
+
+`translate` - Enables translation of the facet values in addition to the facet's
+label. These translations may be defined inside any YAML file in
+`config/locales/` with the following format for each language:
+
+```yaml
+[language]:
+  facet_value:
+    [api_field]:
+      [field_value]: [translation]
 ```
 
+- `language` - Same as above
+- `api_field` - Same as above except nested field names must be written with
+  underscores replacing periods
+    - `field_value` - The text value returned by the API with underscores
+    replacing periods, commas, and spaces
+    - `translation` - The literal translated text for this particular language
+
+As an example, imagine we have a `source` API field we don't want included in
+search filters or translated, and a `category.en` API field we'd like included
```

Comment:
huh?  Are you imagining that we'll use nested language fields in `category` eventually?  If you're just looking for a nested field to use, might I recommend `person.name` ?  Then you could use it as the field that won't be translated (because the names would (probably) stay the same typically), and then use category as the field that will be translated?

Additionally, did you intend to not have the source field show up at all even on the browse pages for the Spanish version?  That's totally reasonable to show in the example, it just wasn't totally clear from the above text that it was intended to be omitted entirely from browse, not just going untranslated / not on the search page.

Something like

```
facets:
  en:
    category:
      label: Category
      flags:
        - search_filter
    person.name:
      label: "People"
  es:
    category:
      label: Categoría
      flags:
        - search_filter
        - translate
    person.name:
      label: Personas
```

Then maybe a little text as well like, "facet fields do not have to match between languages; one language may utilize 5 fields while another only uses 2"

Reply:
I'll switch to a completely made up field (which I'll note somehow) `film.title`.

I wasn't intentionally omitting Source in Spanish, but it is a good concept to demonstrate. :sweat_smile: I'll add a comment in the YAML to point out how the languages can use different facets like that. Thanks!